registered/merge.py

```diff
@@ -33,17 +32,23 @@
 MERGE_EXTENSIONS = ["nde", "plc", "rte", "trp", "pat", "ppat", "blk", "crw", "cal"]
 
 
-def fast_merge(input_filenames, output_filename, extra=""):
+def rename_timepoint(data):
     """
-    Merge files quickly by copying the first one.
+    Replace ';dudly ;' with ';nubn  ;' since the timepoint was renamed
+    but it's not easy to change the ID in HASTUS.
+    """
+    return data.replace(";dudly ;", ";nubn  ;")
+
+
+def merge_and_rename_timepoint(input_filenames, output_filename, extra=""):
+    """
+    Merge files.
     """
-    first_filename = input_filenames[0]
-    shutil.copy(first_filename, output_filename)
 
     with open(output_filename, "a") as output_file:
```

Comment:
You'll also want to open the file as `'w'` (writing) instead of `"a"` (append). We used append before because the first step copied over any existing file with the first input file.